docs/www/quick-start-kubernetes.md

```diff
@@ -126,7 +125,7 @@ to verify that cert-manager is working correcly.
 
     ```
     kubectl apply \
-    -k https://github.com/vectorizedio/redpanda/src/go/k8s/config/crd?ref=$version
+    -k https://github.com/vectorizedio/redpanda/src/go/k8s/config/crd?ref=$VERSION
     ```
 
 4. Install the Redpanda operator on your Kubernetes cluster with:
```

Comment:
NIT: If we add VERSION could we include in the helm install command? The same as in `kubernetes-external-connect.md`